search-service/src/main/kotlin/com/egm/stellio/search/service/EntityOperationService.kt

```diff
@@ -234,4 +246,19 @@ class EntityOperationService(
             sub
         ).bind()
     }
+
+    @SuppressWarnings("UnusedParameter")
+    suspend fun mergeEntity(
```

Comment:
others (`replaceEntity`, ...) are wrapped into an `either` and do a `bind()`. but not sure it is actually needed in this case.